<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mw_filter_image</class>
 <widget class="QWidget" name="mw_filter_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Box Blur</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>radius:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_box">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_apply_box">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Gaussian Blur</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>radius:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_gaussian">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_apply_gaussian">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Rank Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>kernel size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_3x3">
         <property name="text">
          <string>(3x3)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">rbg_kernel_size</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_5x5">
         <property name="text">
          <string>(5x5)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">rbg_kernel_size</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="rb_min">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>min</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">rbg_filter</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_max">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>max</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">rbg_filter</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_median">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>median</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">rbg_filter</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_apply_rank">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rbg_kernel_size"/>
  <buttongroup name="rbg_filter"/>
 </buttongroups>
</ui>
